README.md

```diff
@@ -210,7 +210,7 @@ You can create a Docker image named `io.stargate/jsonapi` using:
 
 Or, if you want to create a native-runnable Docker image named `io.stargate/jsonapi-native` using:
 ```shell script
-./mvnw clean package -Pnative -Dquarkus.native.container-build=true
+./mvnw clean package -Pnative -Dquarkus.native.container-build=true -Dquarkus.container-image.build=true
```

Comment:
doesn't actually build the image without this option

Reply:
may also need to apply this to Stargate repo API readme files

Reply:
I confirmed the READMEs in stargate repo for other APIs have this documented correctly